eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -4,90 +4,22 @@
     "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
-  <!-- AutoRest generates 'final' on interfaces. Remove this suppression when:
```

Comment:
Has this issue already been fixed?

Reply:
Everything in com.microsoft is suppressed... so it doesn't matter.